pkg/etcd/init/init.go

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Nit. I'd personally either keep this file under `clustermesh-apiserver/etcdinit` or put is somewhere under `pkg/kvstore` for consistency with the rest of the kvstore-related code (so that it is also covered by the relevant codeowners).

Reply:
I think I prefer `pkg/` just to keep it separated logically. I could put it under `pkg/kvstore/etcdinit`? `pkg/kvstore/etcd/init`?

Reply:
I'd personally go for `pkg/kvstore/etcdinit` then, because we don't currently have a `pkg/kvstore/etcd` directory with etcd stuff in (it is all in the base `pkg/kvstore` path).

Reply:
Done